packages/graphql/src/translate/where/property-operations/create-aggregate-operation.ts

```diff
@@ -38,7 +38,7 @@ export function createAggregateOperation({
     const relationship = context.relationships.find((x) => x.properties === relationField.properties) as Relationship;
 
     const aggregateStatement = new CypherBuilder.RawCypher((env: CypherBuilder.Environment) => {
-        const varName = env.getVariableId(parentNode);
+        const varName = env.getReferenceId(parentNode);
```

Comment:
This bit is exposed outside of the cypherBuilder due to some hacks for compatibility reasons